node/src/server-modules/GlideFt.ts

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+ */
+
+import { Decoder, DecoderOption, GlideString } from "../BaseClient";
+import { GlideClient } from "../GlideClient";
+import { GlideClusterClient } from "../GlideClusterClient";
+import { Field, FtCreateOptions } from "./GlideFtOptions";
+
+/** Module for Vector Search commands */
+export class GlideFt {
+    /**
+     * Creates an index and initiates a backfill of that index.
+     *
+     * @param client The client to execute the command.
+     * @param indexName The index name for the index to be created.
+     * @param schema The fields of the index schema, specifying the fields and their types.
+     * @param options Optional arguments for the `FT.CREATE` command. See {@link FtCreateOptions}.
+     *
+     * @returns If the index is successfully created, returns "OK".
+     *
+     * @example
+     * ```typescript
+     * // Example usage of FT.CREATE to create a 6-dimensional JSON index using the HNSW algorithm
+     * await GlideFt.create(client, "json_idx1", [{
+     *      type: "VECTOR",
+     *      name: "$.vec",
+     *      alias: "VEC",
+     *      attributes: {
+     *          algorithm: "HNSW",
+     *          type: "FLOAT32",
+     *          dimension: 6,
+     *          distanceMetric: "L2",
+     *          numberOfEdges: 32,
+     *      },
+     *  }], {
+     *      dataType: "JSON",
+     *      prefixes: ["json:"]
+     *  });
+     * ```
+     */
+    static async create(
+        client: GlideClient | GlideClusterClient,
+        indexName: GlideString,
+        schema: Field[],
+        options?: FtCreateOptions,
+    ): Promise<"OK" | null> {
```

Comment:
When it could return `null`? Update if can't:
```suggestion
    ): Promise<"OK"> {
```